internal/ui/ui_glfw.go

```diff
@@ -200,38 +196,96 @@ func initialize() error {
 	return nil
 }
 
-type monitor struct {
+func (u *userInterfaceImpl) setInitMonitor(m *glfw.Monitor) {
+	u.initMonitor = m
+	u.initDeviceScaleFactor = u.deviceScaleFactor(m)
+	// GetVideoMode must be called from the main thread, then call this here and record
+	// initFullscreen{Width,Height}InDIP.
+	v := m.GetVideoMode()
+	u.initFullscreenWidthInDIP = int(u.dipFromGLFWMonitorPixel(float64(v.Width), m))
+	u.initFullscreenHeightInDIP = int(u.dipFromGLFWMonitorPixel(float64(v.Height), m))
+}
+
+// Monitor is a wrapper around glfw.Monitor.
+type Monitor struct {
 	m  *glfw.Monitor
 	vm *glfw.VidMode
 	// Pos of monitor in virtual coords
-	x int
-	y int
+	x      int
+	y      int
+	width  int
+	height int
+	id     int
+	name   string
+}
+
+// Bounds returns the monitor's bounds.
+func (m *Monitor) Bounds() image.Rectangle {
+	return image.Rect(m.x, m.y, m.x+m.width, m.y+m.height)
```

Comment:
I realized that the unit should be 'device-independent-pixels' for consistency with other APIs. Could you convert the values by `u.dipFromGLFWMonitorPixel`?